cmd/fossa/config.go

```diff
@@ -180,6 +190,7 @@ func parseModuleFlag(moduleFlag string) []moduleConfig {
 func initialize(c *cli.Context) (cliConfig, error) {
 	var config = cliConfig{
 		apiKey:   c.String("api_key"),
+		fetcher:  c.String("fetcher"),
```

Comment:
If we don't want users to set this, we should probably not include it as a flag.